### PR TITLE
Allow git user name and email configuration

### PR DESCRIPTION
**Pull request by codingtwinky**
fixes: #817

### CODE REVIEW

**Comment by campersau** [comment]
I am not sure this is what was proposed in #817. I think #817 was about editing the git config file and not the ungit config file. Also this adds another place to put author information.

I was thinking about a settings dialog with user name and email field which are filled from `git config user.name` and `git config user.email` and can be edited and saved via [`git config user.name "abc"`](https://help.github.com/articles/setting-your-username-in-git/) and [`git config user.email "abc@example.com"`](https://help.github.com/articles/setting-your-email-in-git/).

**Comment by codingtwinky** [comment]
I thought about several ideas for this issue but I think it is best to isolate config change for an instance of Ungit.  If you run `git config` any subsequent git operation from any processes within that repo will be changed.  I'm not sure if this is a good idea so I chose this route

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ This project adheres to [Semantic Versioning](http://semver.org/) and
 - Fix for favorites linking in case rootPath is used @sebastianmay [#609](https://github.com/FredrikNoren/ungit/issues/609) and image diffing
 - Limit commit title to 72 characters, the rest is truncated and shown when inspecting the commit
 - Updated file watch logic to closely follow git commands in another process [#283](https://github.com/FredrikNoren/ungit/issues/283)
+- Added away to configure git commit user name and email [#813](https://github.com/FredrikNoren/ungit/issues/813)
```
Comment (Ajedi32) [inline CHANGELOG.md]: away -> a way

**Comment by codingtwinky** [comment]
@campersau fixed it so that it ungit will always send author config string.  Let me know if you guys have any other concerns.

**Comment by codingtwinky** [comment]
For now, commit is the only api that considers user name and email configuration, please let me know if there are any other git commands that accepts author argument.

**Pull request closed without merging**